### PR TITLE
050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF

### PR DESCRIPTION
**Pull request by istoimenova**
Script is part of TS: #1163
Script is common as PRs: #1325, #1327

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 1]
```diff
@@ -0,0 +1,169 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [PTU] [GENIVI] PolicyTableUpdate has NO "certificate" and "ForceProtectedService"=OFF at .ini file
+-- [PTU] [GENIVI] SDL must start PTU for any app except navi right after app successfully request to start first secure service
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Seems "[PTU] [GENIVI] PolicyTableUpdate is valid and brings "certificate"" req-t is missed, isn't it?

Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, done in 460af123d02778595f1ec14c78826966652f6e67

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+-- SDL must start TLSHandshake sequence (by sending TLS Hello to this mobile app)
+--
+-- 1. Used preconditions:
+-- ForceProtectedService is set to OFF in .ini file
+-- Communication app exists in LP, no certificate in module_config
+-- Register and activate application.
+-- Send StartService(serviceType = 7 (RPC))
+-- -> SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- -> SDL should not respond to StartService_request
+-- -> SDL should not process request to HMI
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Please update if sending of SetAudioStreamingIndicator is not required

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+--
+-- 2. Performed steps
+-- Send correct policy file, certificate exists in module_config
+--
+-- Expected result:
+-- 1. SDL sends SDL.OnStatusUpdate(UP_TO_DATE)
+-- 2. SDL process request to HMI
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @istoimenova, Please update if sending of SetAudioStreamingIndicator is not required

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, done.

File: test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua (hunk 3 above)
Comment (istoimenova) [inline test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/050_ATF_P_TC_comm_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua]: @dboltovskyi, done.

**Pull request merged**